.github/workflows/urls-check.yml

```diff
@@ -0,0 +1,22 @@
+name: urls-check
+on:
+  schedule:
+    - cron: '0 4 * * *'
+  # can be run manually on https://github.com/cockpit-project/cockpit/actions
+  workflow_dispatch:
+jobs:
+  urls-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up configuration and secrets
+        run: |
+          printf '[user]\n\tname = Cockpit Project\n\temail=cockpituous@gmail.com\n' > ~/.gitconfig
```

Comment:
This doesn't commit anything, so we don't strictly need a git config here.

Reply:
true, I'll drop it.

Reply:
done